server/py/services/api/crud/projects.py

```diff
@@ -248,23 +235,8 @@ def delete_project_resources(
         )
         self._wait_for_nuclio_project_deletion(name, session, auth_info)
 
-        try:
-            # delete model monitoring resources
-            logger.debug(
-                "Deleting model endpoints resources",
-                project_name=name,
-            )
-            services.api.crud.ModelEndpoints().delete_model_endpoints_resources(
-                project_name=name,
-                db_session=session,
-                model_monitoring_applications=model_monitoring_applications,
-                model_monitoring_access_key=model_monitoring_access_key,
-            )
-        except Exception as exc:
-            logger.warning(
-                "Failed to delete model monitoring resources", project_name=name
-            )
-            raise exc
```

Comment:
The previous behavior

Reply:
Alright I know that we are already catching specific errors for tsdb/kafka (such as `MLRunTSDBConnectionFailureError`) so I guess it's enough for now but we just need to be careful not to break the delete project API due to issues with the external resources